root/etc/wallabag/parameters.template.yml

```diff
@@ -0,0 +1,63 @@
+parameters:
+    database_driver: ${SYMFONY__ENV__DATABASE_DRIVER:-pdo_sqlite}
+    database_host: ${SYMFONY__ENV__DATABASE_HOST:-127.0.0.1}
+    database_port: ${SYMFONY__ENV__DATABASE_PORT:-~}
+    database_name: ${SYMFONY__ENV__DATABASE_NAME:-symfony}
+    database_user: ${SYMFONY__ENV__DATABASE_USER:-root}
+    database_password: ${SYMFONY__ENV__DATABASE_PASSWORD:-~}
+    database_path: "%kernel.root_dir%/../data/db/wallabag.sqlite"
+    database_table_prefix: ${SYMFONY__ENV__DATABASE_TABLE_PREFIX:-wallabag_}
+    database_socket: null
+    database_charset: ${SYMFONY__ENV__DATABASE_CHARSET:-utf8}
+
+    domain_name: ${SYMFONY__ENV__DOMAIN_NAME:-https://your-wallabag-url-instance.com}
```

Comment:
We should replace the default url by something like `itworks.wallabag.org`, see https://github.com/wallabag/wallabag/issues/5890

Reply:
I didn't change the behavior in this PR, Ansible install set the same value.
I think the best solution will be to remove the default value so this Environment Variable is mandatory. This could beak some installations so it has to be documented. Please, do the change in other PR or open an issue to keep the track.

Reply:
Agree'd